### ISSUE TITLE
Translations should use the exact same commands and outputs

### ISSUE THREAD

**Issue by mhmdanas** [nushell/book#94]
For example, in the English version of the introduction, we have this code block:
```
> ls | where size > 1kb
───┬──────────────┬──────┬────────┬────────────
 # │ name         │ type │ size   │ modified
───┼──────────────┼──────┼────────┼────────────
 0 │ Gemfile      │ File │ 1.1 KB │ 3 days ago
 1 │ Gemfile.lock │ File │ 6.9 KB │ 3 days ago
 2 │ LICENSE      │ File │ 1.1 KB │ 3 days ago
 3 │ SUMMARY.md   │ File │ 3.7 KB │ 3 days ago
───┴──────────────┴──────┴────────┴────────────
```

On the other hand, in the Spanish version of the book for example, that code block corresponds to this one:
```
> ls | where size > 4kb
----+----------------+------+----------+----------+----------------+----------------
 #  | name           | type | readonly | size     | accessed       | modified 
----+----------------+------+----------+----------+----------------+----------------
 0  | IMG_1291.jpg   | File |          | 115.5 KB | a month ago    | 4 months ago 
 1  | README.md      | File |          | 11.1 KB  | 2 days ago     | 2 days ago 
 2  | IMG_1291.png   | File |          | 589.0 KB | a month ago    | a month ago 
 3  | IMG_1381.jpg   | File |          | 81.0 KB  | a month ago    | 4 months ago 
 4  | butterfly.jpeg | File |          | 4.2 KB   | a month ago    | a month ago 
 5  | Cargo.lock     | File |          | 199.6 KB | 22 minutes ago | 22 minutes ago
```

Disregarding the commands (which are exactly the same, sans the file size), the outputs are completely different. Using the same commands, outputs, etc would make future translations easier.

**Comment by jzaefferer** [comment]
I think this is mostly an issue of the english intro being up-to-date, and the translations still needing the same updates.

Would you like to help out with that? See https://github.com/nushell/book/pull/77 for the PR that updated the english introduction.

**Comment by thegedge** [comment]
It would be amazing if the examples could be shared (so they get "included" into the text) and also regularly tested with the latest version of nu (so they don't become stale). That's a whole project on its own though 🙂

**Comment by mhmdanas** [comment]
@jzaefferer I'm happy to help! Where should I start? (I want to point out I don't know any of the translated languages except English.)

**Comment by mhmdanas** [comment]
About the testing part, this might be useful: https://github.com/budziq/rust-skeptic

**Comment by jzaefferer** [comment]
For the sharing/reusing, I'm sure Jekyll has some feature. So the process could be: Extract a code example from the English intro into a separate file, include it in the English version, and use it for the translations.

Since nu itself is only in English (right?), the code examples could go into a new folder in the project root. Maybe "snippets"?

**Comment by mhmdanas** [comment]
@jzaefferer yeah, I found it: https://jekyllrb.com/docs/includes/#including-files-relative-to-another-file

**Comment by mhmdanas** [comment]
I opened #99, which moves all code blocks in `en/installation.md` into `_includes/installation`.

**Comment by mhmdanas** [comment]
I can't find the commits for #99 and #105 in `master`. Can somebody explain to me what's happening?

**Comment by thegedge** [comment]
@mhmdanas I think something messed up recently when we tried to switch from `master` to `main`. I've managed to get us back to #110, which means those two commits are back. We lost #111 and #112, but we'll cherry pick those changes back.

**Comment by mhmdanas** [comment]
I'm a little bit confused here. I want to contribute something, but should I merge into `master` or `main`?

**Comment by thegedge** [comment]
@mhmdanas We currently still use `master` in this repo because of GitHub pages, but note that we will eventually switch to `main`.
